js/mainScript.js

```diff
@@ -35,6 +35,7 @@ function handleCsvParsing(evt) {
 }
 
 // took this one here: https://ourcodeworld.com/articles/read/189/how-to-create-a-file-and-generate-a-download-with-javascript-in-the-browser-without-a-server
+// TODO(supervanya): get one from npm instead of manual implementation
 function download(filename, text) {
   var element = document.createElement("a");
   element.setAttribute(
```

Comment:
I took this function from the internet by copy-pasting. Industry practice for most cases is to use package distribution platforms. It allows you to separate concerns and focus on what's at hand and trust other developers to solve a problem that is not your immediate concern.
For python it is [PiPy](https://pypi.org/), for javascript it is [NPM](npmjs.com). 

What's cool is you can kinda see how each language is developing by glancing at total [package counts for each system](http://www.modulecounts.com/). 
![image](https://user-images.githubusercontent.com/12336038/151105332-3abfed81-ab6a-4ff0-bc1e-a1b125c81e91.png)

Not a bad time to learn JS?